app/routes/endemics/declaration.js

```diff
@@ -151,11 +150,10 @@ module.exports = [
 
         return h.view(endemicsConfirmation, {
           reference: newApplicationReference,
-          isNewUser:
-            userType.NEW_USER === application.organisation.userType &&
+          isNewUser: userType.NEW_USER === application.organisation.userType &&
             !oldWorldRejectedAgreement10months.isExistingUserRejectedAgreementWithin10months,
-          hasRejectedApplicationInPastTenMonths:
-            oldWorldRejectedAgreement10months.isExistingUserRejectedAgreementWithin10months,
+          hasRejectedApplicationInPastTenMonths: oldWorldRejectedAgreement10months.isExistingUserRejectedAgreementWithin10months,
+          hasOldWorldReadyToPayWithinTenMonths: oldWorldReadyToPayWithin10Months.isExistingUserReadyToPayAgreementWithin10months,
```

Comment:
Instead of new flags could we instead put these onto the userType object?

Reply:
ok